src/System.Runtime.Extensions/src/System/Environment.Unix.cs

```diff
@@ -36,6 +37,7 @@ public static partial class Environment
             }
             return results;
         });
+        private static readonly bool IsMac = Interop.Sys.GetUnixName() == "OSX";
```

Comment:
Not sure we need to revisit at this point but it is an example of why I like the new RuntimeInformation APIs better because a platform can be both Unix and Mac.